NDTensors/src/lib/Expose/test/runtests.jl

```diff
@@ -184,7 +184,11 @@ using NDTensors.GPUArraysCoreExtensions: cpu
   copyto!(expose(y), expose(x))
   @allowscalar begin
     @test y == x
-    @test copy(x) == x
+    if dev == NDTensors.mtl
+      @test_broken copy(x) == x
```

Comment:
Does `copy(expose(x))` work? Or is that not needed since we don't `copy` this particular wrapped type in the code anyway?

Reply:
Yes the expose version of copy does work with metal so I switched it over to use that one. Thanks!